_includes/sidebar.html

```diff
@@ -101,9 +101,10 @@
 
       <h3>Developer Program</h3>
       <ul>
-        <li><a href="/api">The API</a></li>
+        <li class="is-overview"><a href="/user/developer/">Overview</a></li>
+        <li><a href="http://developer.travis-ci.com/">API V3</a></li>
```

Comment:
it's on .com and .org but maybe put .org here as we link to the explorer on .org below...